web/src/components/apps/AppSnapshots.jsx

```diff
@@ -154,37 +178,42 @@ class AppSnapshots extends Component {
       restoringSnapshot,
       snapshotToRestore,
       restoreErr,
-      restoreErrorMsg
+      restoreErrorMsg,
+      configuration,
+      hideCheckVeleroButton
     } = this.state;
-    const { app, snapshots, snapshotSettings } = this.props;
+    const { app, snapshots, snapshotSettings, isVeleroInstalled } = this.props;
     const appTitle = app.name;
-    const veleroInstalled = true;
 
-    if (snapshots?.loading ||snapshotSettings?.loading) {
+    if (snapshots?.loading || snapshotSettings?.loading || isVeleroInstalled?.loading) {
```

Comment:
I think the loader for while we're checking if Velero is installed should replace the button, not the full page loader. Similar to the loader that shows when you save the config.

Reply:
How it can replace the button if we are using the same query to show the "Velero not installed" page. https://github.com/replicatedhq/kotsadm/pull/1210/files#diff-3185d5cdf16b8820577989008f5d3541R197